src/ResourceManager/KeyVault/Commands.KeyVault/help/Undo-AzureRmKeyVaultRemoval.md

```diff
@@ -20,7 +20,7 @@ Undo-AzureRmKeyVaultRemoval [-VaultName] <String> [-ResourceGroupName] <String>
 
 ### InputObject
 ```
-Undo-AzureRmKeyVaultRemoval [-InputObject] <PSDeletedKeyVault> [-ResourceGroupName] <String> [-Tag <Hashtable>]
```

Comment:
@maddieclayton `ResourceGroupName` was removed, but still shows up in the example and in the list of parameters below

Reply:
@maddieclayton oops ignore this